project/project-router.js

```diff
@@ -0,0 +1,52 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3'
+    }
+}
+
+const db = knex(knexConfig);
+
+router.get('/', (req, res) => {
+    db('projects')
+    .then(projects => {
+        res.status(200).json(projects)
+    })
+    .catch(err => {
+        res.status(500).json(err)
+    })
+}); 
+
+router.post('/', async (req, res) => {
+    try {
+        const project = await db('projects').insert(req.body)
+        res.status(201).json(project)
+    } catch (error) {
+        res.status(500).json({ error: 'There was an error posting that!' })
+    }
+});
+
+
+router.get('/project/:id', (req, res) => {
+    const { id } = req.params;
+        db('projects')
+           .where({ id: id })
```

Comment:
Good work on the stretch endpoint that lists project actions, here you can simplify to `.where({ id })` and you should always aim to include error handling in case the ID does not exist (404 status).